luggage_announcements.flexslider_default_preset.inc

```diff
@@ -16,8 +16,6 @@ function luggage_announcements_flexslider_default_presets() {
   $preset->name = 'announcemant_slider';
   $preset->title = 'announcemant slider';
   $preset->theme = 'classic';
-  $preset->imagestyle_normal = 'announcement_full';
```

Comment:
These may conflict with PR #22 as that also deletes these presets. See LUGG-770. Just something to be aware of.